keras_nlp/layers/transformer_decoder.py

```diff
@@ -252,6 +252,8 @@ def call(
                 at which to update the `cross_attention_cache`. Usually, this is
                 either `0` (compute the entire `cross_attention_cache`), or
                 `None` (reuse a previously computed `cross_attention_cache`).
+             use_causal_mask: bool, defaults to True. If true, causal mask
+                (masking out future input) is applied on the decoder sequence.
```

Comment:
naming and documentation from previous version where this was in: https://github.com/keras-team/keras-nlp/blob/cb0fa028971475879911ddf042a1473037775ee6/keras_nlp/layers/transformer_decoder.py#L191-L192

Reply:
this lgtm, and matches the multi-head attention layer in core Keras